src/open_inwoner/configurations/models.py

```diff
@@ -664,8 +663,11 @@ def get_accent_color(self):
         return hex_to_hsl(self.accent_color)
 
     @property
-    def get_ordered_flatpages(self):
-        return self.flatpages.order_by("ordered_flatpages")
+    def get_ordered_cms_pages(self):
```

Comment:
```suggestion
    def ordered_cms_pages(self):
```
The imperative verb suggests it should be a method: I would either rename or remove `@property` (I am aware this pattern was introduced earlier, but let's try and break it :).

Reply:
Was tempted to do this but shied away for fear of making things more confusing. If we're on the same page, though, let's go for it.